src/services/ui-infra/serverless.yml

```
@@ -283,7 +283,7 @@ resources:
     ApplicationEndpointUrl:
       Type: AWS::SSM::Parameter
       Properties:
-        Name: /${self:custom.stage}/ui/application_endpoint
+        Name: /${self:service}/${self:custom.stage}/application_endpoint
```

```suggestion
        Name: /${self:custom.stage}/${self:service}/application_endpoint
```
Can we flip these around like this?

---

We can do all things. Done!